app/includes/layout.njk

```diff
@@ -38,9 +38,17 @@
 
     {% include 'includes/header.njk' %}
 
-      <main class="nhsuk-main-wrapper" id="maincontent">
-        {% block body %}{% endblock %}
-      </main>
+    {% block mainOpen %}
+      <div class="nhsuk-width-container">
+        <main class="nhsuk-main-wrapper" id="maincontent">
+          {% endblock %}
+
+          {% block body %}{% endblock %}
+
+          {% block mainClose %}
+        </main>
+      </div>
+    {% endblock %}
```

Comment:
I'm not sure about this. Can you explain a bit about why you are having to do this and what's going on here?

Reply:
I'm guessing we need different classes in L42 and L43 in some components so can swap them in this way?